.github/workflows/loadtest.yaml

```diff
@@ -41,16 +38,16 @@ jobs:
 
       - name: Creating database hlgraphql
         run: psql "postgres://postgres:password@localhost:5432/rollupsdb?sslmode=disable" -c "CREATE DATABASE hlgraphql WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'en_US.UTF-8' LC_CTYPE = 'en_US.UTF-8';"
-      
+
       - name: Using Postgres DB
-        run : |
+        run: |
           export POSTGRES_HOST=localhost
           export POSTGRES_PORT=5432
           export POSTGRES_DB=hlgraphql
           export POSTGRES_USER=postgres
           export POSTGRES_PASSWORD=password
-          nohup ./cartesi-rollups-hl-graphql -d --raw-enabled --high-level-graphql --graphile-disable-sync --db-implementation=postgres --db-raw-url=postgres://postgres:password@localhost:5432/rollupsdb?sslmode=disable --disable-devnet &
-          
+          nohup ./cartesi-rollups-hl-graphql -d --raw-enabled --graphile-disable-sync --db-implementation=postgres --db-raw-url=postgres://postgres:password@localhost:5432/rollupsdb?sslmode=disable &
```

Comment:
we can use only the env vars to run this test

Reply:
take a look at https://github.com/Calindra/cartesi-rollups-hl-graphql/pull/18

Reply:
Resolved https://github.com/Calindra/cartesi-rollups-hl-graphql/pull/20/commits/0d69e4f756babd0303aa2189070d6c7bb9834a21